### PR TITLE
Split page into different methods

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Broken anchor tags in this doc. Decided to split the docs into the different methods rather than just fix the tags.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/50

### CODE REVIEW

File: timescaledb/how-to-guides/migrate-data/import-csv.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Import data into TimescaleDB from .csv
```
Comment (jonatas) [inline timescaledb/how-to-guides/migrate-data/import-csv.md:1]: I see some bugs related to highlight close to headers. Maybe we should have a short breath between the elements?
Check #53.
```suggestion
# Import data into TimescaleDB from .csv

```

**Review by jonatas — APPROVED** [review]
Very good @Loquacity! I think it also will improve the SEO of the pages.

File: timescaledb/how-to-guides/migrate-data/different-db.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+To migrate your database from PostgreSQL to TimescaleDB, you
+need `pg_dump` for exporting your schema and data.
+
+Migration falls into three main steps:
+
+1. Copy over the database schema and choose which tables will become
+hypertables (i.e., those that currently have time-series data).
+1. Backup data to comma-separated values (CSV).
+1. Import the data into TimescaleDB
```
Comment (jonatas) [inline timescaledb/how-to-guides/migrate-data/different-db.md:16]:
```suggestion
1. Copy over the database schema and choose which tables will become
hypertables (i.e., those that currently have time-series data).
2. Backup data to comma-separated values (CSV).
3. Import the data into TimescaleDB
```

File: timescaledb/how-to-guides/migrate-data/import-csv.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+<highlight type="tip">
+First make sure that you have properly [installed](/timescaledb/latest/how-to-guides/install-timescaledb/)
+**AND [setup](/timescaledb/latest/how-to-guides/install-timescaledb/post-install-setup/)** TimescaleDB
+within your PostgreSQL instance.
+</highlight>
+
+If you have data stored in an external `.csv` file, you can import it into TimescaleDB:
+
+1. Create a new empty table with the same schema as the data file and convert the table to a hypertable.
+2.  Insert the data from the file.
```
Comment (jonatas) [inline timescaledb/how-to-guides/migrate-data/import-csv.md:11]:
```suggestion
2. Insert the data from the file.
```

File: timescaledb/how-to-guides/migrate-data/import-csv.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+InfluxDB to TimescaleDB. Anyone who is currently running an InfluxDB instance
+can migrate their workload to TimescaleDB with a single command: `outflux migrate`.
+You must also have [TimescaleDB installed][installed] and a means to connect to it.
+
+With Outflux, users can pipe exported data directly into TimescaleDB.
+Outflux manages schema discovery, validation, and creation.
+
+For more information on how to get started, please follow [this tutorial][outflux-tutorial].
+
+
```
Comment (jonatas) [inline timescaledb/how-to-guides/migrate-data/import-csv.md:83]:
```suggestion

```

File: timescaledb/how-to-guides/migrate-data/different-db.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+# Migrating from a Different PostgreSQL Database
```
Comment (jonatas) [inline timescaledb/how-to-guides/migrate-data/different-db.md:1]: Probably we should normalize the headers as we did on #39. WDYT?

```suggestion
# Migrating from a different PostgreSQL database

```

**Review by jonatas — APPROVED** [review]
Apart from normalizing the headers, everything is better! thank you for the improvements!

File: timescaledb/how-to-guides/migrate-data/different-db.md (hunk 5 above)
Comment (Loquacity) [reply timescaledb/how-to-guides/migrate-data/different-db.md:1]: yep, agreed 👍

File: timescaledb/how-to-guides/migrate-data/different-db.md (hunk 2 above)
Comment (Loquacity) [reply timescaledb/how-to-guides/migrate-data/different-db.md:16]: Doesn't this just create maintenance overhead, and risk numbering errors?

Comment (jonatas) [reply timescaledb/how-to-guides/migrate-data/different-db.md:17]: Sorry, I didn't verify if gatsby fixes it for us or how the numbers will get rendered. I just thought it was not observed. Is it going to be automatic?

Comment (Loquacity) [reply timescaledb/how-to-guides/migrate-data/different-db.md:17]: It's standard markup, and it does appear to be working properly:

image

http://docs-v3-dev.timescale.com.s3-website-us-east-1.amazonaws.com/docs-migrate-links-lana/timescaledb/migrate-links-lana/how-to-guides/migrate-data/different-db/#copying-schema-amp-setting-up-hypertables

**Pull request merged**